web/src/main/java/com/epam/esm/configs/SpringConfig.java

```diff
@@ -0,0 +1,81 @@
+package com.epam.esm.configs;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ComponentScan("com.epam.esm")
+@PropertySource("classpath:mysql.properties")
+@EnableTransactionManagement
+@EnableWebMvc
+public class SpringConfig {
```

Comment:
propose to split since it's not a configuration of the spring, but it mostly the app config (or parts of app like db) using spring

Reply:
ok.